docs/source/tutorials/mmlab_models_usage_guide.md

```diff
@@ -0,0 +1,67 @@
+# Use mmlab's models in EasyCV
+
+**We only support mmdet's models and do not support other series in mmlab and other modules such as transforms, dataset api, etc. are not supported either.**
+
+The models module of EasyCV is divided into four modules: `backbone`, `head`, `neck`, and `model`.
+
+So we support the models combination of EasyCV and mmlab from these four levels.
+
+**We will not adapt the other apis involved in these four levels modules, we package the entire api for use.**
+
+> **Note: **
+>
+> **If you want to combine the models part of mmdet and easycv, please pay attention to the compatibility between the apis, we do not guarantee that the api of EasyCV and mmlab are compatible.**
+
+Take the `MaskRCNN` model as an example, please refer to [mask_rcnn_r50_fpn.py](https://github.com/alibaba/EasyCV/tree/master/configs/detection/mask_rcnn/mask_rcnn_r50_fpn.py). Except for the backbone, other parts in this model are all mmdet apis.
+
+The framework of `MaskRCNN` can be divided into the following parts from the `backbone`, `head`, `neck`, and `model` levels：
+
+- backbone: `ResNet`
+
+- head：`RPNHead`,	`StandardRoIHead`
+
+- neck: `FPN`
+
+- model: `MaskRCNN`
+
+The configuration adapt for mmdet is as follows:
+
+```python
+mmlab_modules = [
+    dict(type='mmdet', name='MaskRCNN', module='model'),
+    # dict(type='mmdet', name='ResNet', module='backbone'), # comment out, use EasyCV ResNet
+    dict(type='mmdet', name='FPN', module='neck'),
+    dict(type='mmdet', name='RPNHead', module='head'),
+    dict(type='mmdet', name='StandardRoIHead', module='head'),
+]
+```
+
+> Parameters:
+>
+> 	- type: the name of the open source, only `mmdet` is supported
+> 	- name:  the name of api
+> 	- Module: The name of the module to which the api belongs, only  `backbone`,`head`,`neck`,`model` are supported.
+
+In this configuration , the `head`, `neck`, and `model` parts specify the type as `mmdet`, except for `backbone`.
```

Comment:
for other models in mmdetection, what modules should user declare?  we should a give a brief description, and provide more config examples in the future

Reply:
For details, you can refer to the doc and case in line9,lin61,line65